app/controllers/users_controller.rb

```diff
@@ -8,7 +8,7 @@ def create
     if @user.save
       session[:user_id] = @user.id
 
-      send_welcome_email(@user)
+      # send_welcome_email(@user)
```

Comment:
As a group we decided to comment this code out so the app will run on heroku. We can go back and uncomment once we set up an email provider!